fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckCards.kt

```diff
@@ -0,0 +1,149 @@
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private val cardShape = RoundedCornerShape(8.dp)
+private val defaultCardElevation = 5.dp
+
+/**
+ * A card container for review quality check UI that can be expanded and collapsed.
+ *
+ * @param title The title of the card.
+ * @param modifier Modifier to be applied to the card.
+ * @param content The content of the card.
+ */
+@Composable
+fun ReviewQualityCheckExpandableCard(
+    title: String,
+    modifier: Modifier = Modifier,
+    content: @Composable () -> Unit,
+) {
+    ReviewQualityCheckCard(modifier = modifier) {
+        var isExpanded by remember { mutableStateOf(false) }
+
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier
+                .fillMaxWidth()
+                .clickable {
+                    isExpanded = isExpanded.not()
+                },
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Text(
+                text = title,
+                color = FirefoxTheme.colors.textPrimary,
+                style = FirefoxTheme.typography.headline8,
+            )
+
+            val chevronDrawable = if (isExpanded) {
+                R.drawable.ic_chevron_up
+            } else {
+                R.drawable.ic_chevron_down
+            }
+
+            Icon(
+                painter = painterResource(id = chevronDrawable),
+                contentDescription = null,
+                tint = FirefoxTheme.colors.iconPrimary,
+                modifier = Modifier.size(20.dp),
```

Comment:
We shouldn't have to ever set the size of the Icons. Seeing the 20dp here indicates to me that we aren't using the correct drawable. 

See: 
https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/20/xml/ic_chevron_down_20.xml 
https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/24/xml/chevron_down_24.xml

Reply:
Updated to use the 20 variant, thanks for the links!